### ISSUE TITLE
Cannot mix projected and non-projected properties

### ISSUE THREAD

**Issue by atrauzzi** [ChilliCream/graphql-platform#6041]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Product

Hot Chocolate

### Describe the bug

I have the following:

```csharp
    [GraphQLIgnore]
    internal IList<LockConfiguration> DomainLockConfiguration { get; init; } = new List<LockConfiguration>();

    [IsProjected(false)]
    public IList<string> LockSchemes => DomainLockConfiguration.Select((configuration) => configuration switch
    {
        PaymentLockConfiguration => "payment",
        _ => throw new("Unsupported lock scheme encountered."),
    }).ToList();
```

But whenever I include `lockSchemes` in my queries, it always comes out as an empty array and the getter is never run.

### Steps to reproduce

Create a computed property on a type to be serialized.

### Version

13

**Comment by glen-84** [comment]
@PascalSenn Can this be closed as a duplicate of #2575?

**Comment by glen-84** [comment]
Closing as a duplicate of #2575.
